### PR TITLE
SUBMARINE-1133. Connect API for CLI Notebooks

### PR DESCRIPTION
**Pull request by atosystem**
### What is this PR for?
Implement
```bash=
submarine list notebook
submarine get notebook <id>
submarine delete notebook <id>
```

### What type of PR is it?
[Feature]

### Todos
None

### What is the Jira issue?
https://issues.apache.org/jira/browse/SUBMARINE-1133

### How should this be tested?
python e2e tests are implemented

### Screenshots (if appropriate)
None

### Questions:
* Do the license files need updating? No
* Are there breaking changes for older versions? No
* Does this need new documentation? Yes

### CODE REVIEW

File: submarine-sdk/pysubmarine/submarine/cli/notebook/command.py [hunk 1]
```diff
@@ -15,24 +15,132 @@
...
+from rich.panel import Panel
+from rich.table import Table
+
+from submarine.cli.config.config import loadConfig
+from submarine.client.api.notebook_client import NotebookClient
+from submarine.client.exceptions import ApiException
+
+submarineCliConfig = loadConfig()
+if submarineCliConfig is None:
+    exit(1)
```
Comment (pingsutw) [inline submarine-sdk/pysubmarine/submarine/cli/notebook/command.py:33]: It's better to raise an exception with error messages here.

File: submarine-sdk/pysubmarine/submarine/cli/notebook/command.py [hunk 2]
```diff
@@ -15,24 +15,132 @@
...
+TIMEOUT = 30  # sec
 
 
 @click.command("notebook")
 def list_notebook():
     """List notebooks"""
-    click.echo("list notebook!")
+    COLS_TO_SHOW = ["Name", "ID", "Environment", "Resources", "Status"]
+    console = Console()
+    # using user_id hard coded in SysUserRestApi.java
```
Comment (pingsutw) [inline submarine-sdk/pysubmarine/submarine/cli/notebook/command.py:49]: add the link in the comment. https://github.com/apache/submarine/blob/5040068d7214a46c52ba87e10e9fa64411293cf7/submarine-server/server-core/src/main/java/org/apache/submarine/server/workbench/rest/SysUserRestApi.java#L228

**Review by pingsutw — COMMENTED** [review]
Thank you, this is awesome. 
some suggestions in the comment.

File: submarine-sdk/pysubmarine/submarine/cli/notebook/command.py (hunk 1 above)
Comment (atosystem) [reply submarine-sdk/pysubmarine/submarine/cli/notebook/command.py:33]: @pingsutw I have already raised the error message in `loadConfig`

https://github.com/apache/submarine/blob/efb4e7c578f9ee5188414b2140ab7b4a9de1ec92/submarine-sdk/pysubmarine/submarine/cli/config/config.py#L103-L114

Do I have to change the place to raise the error?

File: submarine-sdk/pysubmarine/submarine/cli/notebook/command.py (hunk 2 above)
Comment (atosystem) [reply submarine-sdk/pysubmarine/submarine/cli/notebook/command.py:49]: Got it

**Review by pingsutw — APPROVED** [review]
LGTM, again this looks awesome.

**Pull request closed without merging**